datapreview.tex

```diff
@@ -60,28 +60,107 @@ \subsubsection{Data Preview 0.3}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_rubin_sv_95_-25_r.pdf}
+        \caption{Rubin SV 95 -25}
+        \label{fig:img7}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_seagull_r.pdf}
+        \caption{Seagull}
+        \label{fig:img8}
+    \end{subfigure}
+     \caption{Cumulative imaging depth expressed in terms of the $S/N=5$ limiting magnitude for unresolved sources for seven ComCam Deep Drilling Fields.}
+     \label{fig:dp1_fields_psf_maglim}
+\end{figure}
+
+The  processing and preparation of ComCam data for DP1 will take place during the first half of 2025, with an expected DP1 release data in June or July 2025. 
+The planned data products for DP1 are presented in table~ \ref{tab:data-preview-summary}. 
+Data products that are marked as a stretch goal, DRP ForcedSource Catalogs, Difference Images and DIA Catalogs,  will not be confirmed until close to the DP1 release date. 
+DP1 will not contain any DRP  SSP data products in order to focus on readiness for LSSTCam and on providing high quality live LSSTCam SSP Prompt data products as part of the ramp up of Alert Production starting roughly contemporaneously with the DP1 release.
+
+
+%%% rehash %%%
+ComCam observations that are expected to be included in Data Preview 1.
+
+Keep in mind that this is “expected” in a general sense.
+Exact image counts and coadd depths may change.
+
+
+For example, it is conceivable that a few individual exposures might not, in the end, meet the quality control standard of being “science-grade, astronomically useful” images.
 
-In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
-At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects. 
-Table~\ref{tab:data-preview-summary} presents a current best idea of  the data products that might comprise DP1. 
-
-A final decision on the dataset on which DP1 will be based will be taken in February 2024 following the Project's confirming decision about ComCam on-sky data-taking (\S~\ref{ssec:commissioning-schedule})
 
 \subsection{Data Preview 2}
 \label{sec:dp2}
 
-Data Preview 2 will serve a full consistent reprocessing of all data collected as part of the LSSTCam Science Validation Surveys (SV Surveys) together with any other science-grade commissioning images taken throughout the Science Optimization phase of commissioning, including the DP1 data. 
+As the survey begins, all science-grade data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, is reprocessed to produce the final Data Preview, DP2, which will be released 6 months following the start of operations.
```

Comment:
Do we want to be more circumspect here - and not promise _all_ science-grade data from system optimization? No worries if folks are happy with processing and supporting the full set.

Reply:
I think this fine as we specify  "science-grade". If during the DP1 reprocessing, any data falls below our SDQA standards, it will not be "science-grade"

Reply:
We really do want to provide all the data we can, so will only omit data that falls below a quality threshold